endpoints/remotehost/remotehost

```diff
@@ -282,280 +320,366 @@ function remotehost_req_check() {
-            (( total_rc += ${?} ))
-        elif [ "${os_runtime}" == "podman" ]; then
-            container_name="${endpoint_label}_${run_id}_${this_cs_label}_${os_runtime}"
-            do_ssh ${user}@${host} podman logs ${container_name} > "${engine_logs_dir}/${log_file}"
-            (( total_rc += ${?} ))
+            remote_cs_log_file="$remote_logs_dir/$log_file"
+            do_scp "$user@$host" "$remote_cs_log_file" "" $engine_logs_dir
+            (( total_rc += $? ))
+        elif [ "$os_runtime" == "podman" ]; then
+            container_name="${run_id}_${engine_label}"
+            do_ssh $user@$host podman logs $container_name > "$engine_logs_dir/$log_file"
+            (( total_rc += $? ))
+        else
+            echo "WARNING: os_runtime $os_runtime not supported"
         fi
     done
 
-    if [ "${delete_remote_dir}" == "1" -a ${total_rc} == 0 ]; then
+    if [ "$delete_remote_dir" == "1" -a $total_rc == 0 ]; then
         echo "Deleting remote directory:"
-        do_ssh ${user}@${host} /bin/rm -rv ${remote_dir}
-        (( total_rc += ${?} ))
+        do_ssh $user@$host /bin/rm -rv $remote_dir
+        (( total_rc += $? ))
     fi
 
-    return ${total_rc}
+    return $total_rc
 }
 
 function endpoint_remotehost_cleanup() {
     local log_rc cleanup_rc
 
+    # globals
+    # <none>
+
     move_remotehost_logs
-    log_rc=${?}
+    log_rc=$?
 
     cleanup_osruntime
-    cleanup_rc=${?}
+    cleanup_rc=$?
 
-    return $(( ${log_rc} + ${cleanup_rc} ))
+    return $(( $log_rc + $cleanup_rc ))
 }
 
 function endpoint_remotehost_sysinfo() {
-    local local_dir packrat_rc
+    local local_dir="$endpoint_run_dir/sysinfo"
+    local packrat_rc=0
 
-    packrat_rc=0
+    # globals used:
+    # endpoint_run_dir user host
 
-    local_dir="${endpoint_run_dir}/sysinfo"
-    mkdir ${local_dir}
+    mkdir $local_dir
 
-    do_scp "" "${packrat_dir}/packrat" "${user}@${host}" "${remote_dir}"
-    (( packrat_rc += ${?} ))
+    do_scp "" "$packrat_dir/packrat" "$user@$host" "$remote_dir"
+    (( packrat_rc += $? ))
 
-    do_ssh ${user}@${host} mkdir -p ${remote_dir}/sysinfo
-    (( packrat_rc += ${?} ))
+    do_ssh $user@$host mkdir -p $remote_dir/sysinfo
+    (( packrat_rc += $? ))
 
-    do_ssh ${user}@${host} ${remote_dir}/packrat ${remote_dir}/sysinfo
-    (( packrat_rc += ${?} ))
+    do_ssh $user@$host $remote_dir/packrat $remote_dir/sysinfo
+    (( packrat_rc += $? ))
 
-    do_scp "${user}@${host}" "${remote_dir}/sysinfo/packrat-archive" "" "${local_dir}"
-    (( packrat_rc += ${?} ))
+    do_scp "$user@$host" "$remote_dir/sysinfo/packrat-archive" "" "$local_dir"
+    (( packrat_rc += $? ))
 
-    do_ssh ${user}@${host} rm -Rf ${remote_dir}/packrat ${remote_dir}/sysinfo
-    (( packrat_rc += ${?} ))
+    do_ssh $user@$host rm -Rf $remote_dir/packrat $remote_dir/sysinfo
+    (( packrat_rc += $? ))
 
-    return ${packrat_rc}
+    return $packrat_rc
+}
```

Comment:
Yes, I am.  IMO the gratuitous use of {}'s assumes a lack of comprehension of the bash syntax by the reader.

Reply:
I'm not complaining, just merely commenting.  I think we have discussed this somewhat before, but IMHO there are places where the use of the `{}` really help in understanding what is going on, especially when you reviewing code that you the reader didn't write so the variable names may not be familiar -- ie. does `$foo_bar` mean `${foo}_bar` or does it mean `${foo_bar}`.  Then, in order to avoid worrying about whether I should be using the `{}` or not in individual situations I just put myself in the habit of always using them so that it's always explicit.